app/api/bread/schema.py

```diff
@@ -0,0 +1,35 @@
+from marshmallow import fields, ValidationError
+
+from app import ma
+from app.models import BreadType
+
+
+def validate_not_none(value):
+    """
+    Validates whether the given value is None.
+    """
+    if value is None:
+        raise ValidationError('Items contains an unknown bread type')
+
+
+class BreadTypeSchema(ma.Schema):
+    name = fields.String()
+    price = fields.Integer()
+
+
+class OrderSchema(ma.Schema):
```

Comment:
bereken ook de prijs voor een order!

Reply:
Momenteel gebeurt het in de frontend, maar het is waarschijnlijk een goed idee die al in de backend te berekenen.

Reply:
Doet er niet zoveel toe, maar dat maakt de frontend wat minder complex ..